pillar/top.sls

```diff
@@ -6,5 +6,6 @@ base:
     - ip_addrs
     - fqdn
     - kubelet
+    - availability_zone
```

Comment:
It's used in all machines, including masters.

Also, the convention does not appear to be `_`, as you'll see `ip_addrs` directly above this.